query-engine/driver-adapters/js/driver-adapter-utils/src/result.ts

```diff
@@ -0,0 +1,8 @@
+import { Result } from "./types";
```

Comment:
Really wished for `ok` constructor function today, I guess it will go to this module once it lands 😄

Reply:
Might be too late with that, but done.